### PR TITLE
Build and update Arch Mondoo Metapackage

### PR DESCRIPTION
**Pull request by benr**

### CODE REVIEW

**Comment by benr** [comment]
Workflow tested successfully: https://aur.archlinux.org/packages/mondoo

File: packages/archlinux/mondoo/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+pkgname=mondoo
+orignalVersion="2.2.3"
+pkgver="${orignalVersion/-/_}"
+pkgrel=1
+pkgdesc="Mondoo Client CLI for the Mondoo Policy as Code Platform"
+url="https://mondoo.com"
+license=('custom')
+source=(
+    "https://releases.mondoo.com/mondoo/${orignalVersion}/mondoo_${orignalVersion}_linux_amd64.tar.gz"
```
Comment (jaym) [inline packages/archlinux/mondoo/PKGBUILD:13]: this isn't a real file:
```
❯ makepkg
==> Making package: mondoo 2.2.3-1 (Wed Jan 31 14:55:46 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading mondoo_2.2.3_linux_amd64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   127    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
==> ERROR: Failure while downloading https://releases.mondoo.com/mondoo/2.2.3/mondoo_2.2.3_linux_amd64.tar.gz
    Aborting...
    ```

Comment (benr) [reply packages/archlinux/mondoo/PKGBUILD:13]: File removed from generator, it's no needed.

**Comment by benr** [comment]
All fixed now, tested in an Arch Container:

```
[benr@c8e07418b50e ~]$ yay -S --noconfirm  mondoo
AUR Explicit (1): mondoo-10.1.3-1
:: PKGBUILD up to date, skipping download: mondoo
  1 mondoo                                   (Installed) (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
  1 mondoo                                   (Installed) (Build Files Exist)
==> Diffs to show?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
==> Making package: mondoo 10.1.3-1 (Thu Feb  1 01:54:31 2024)
==> Retrieving sources...
  -> Found LICENSE.html
  -> Found OSS-LICENSES.tar.xz
  -> Found mondoo.service
  -> Found mondoo.sh
==> WARNING: Skipping verification of source file PGP signatures.
==> Validating source files with sha256sums...
    LICENSE.html ... Passed
    OSS-LICENSES.tar.xz ... Passed
    mondoo.service ... Passed
    mondoo.sh ... Passed
:: (1/1) Parsing SRCINFO: mondoo
==> Making package: mondoo 10.1.3-1 (Thu Feb  1 01:54:32 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found LICENSE.html
  -> Found OSS-LICENSES.tar.xz
  -> Found mondoo.service
  -> Found mondoo.sh
==> Validating source files with sha256sums...
    LICENSE.html ... Passed
    OSS-LICENSES.tar.xz ... Passed
    mondoo.service ... Passed
    mondoo.sh ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting OSS-LICENSES.tar.xz with bsdtar
==> Sources are ready.
==> Making package: mondoo 10.1.3-1 (Thu Feb  1 01:54:33 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "mondoo"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: mondoo 10.1.3-1 (Thu Feb  1 01:54:34 2024)
==> Cleaning up...
loading packages...
warning: mondoo-10.1.3-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)  Old Version  New Version  Net Change

mondoo       10.1.3-1     10.1.3-1       0.00 MiB

Total Installed Size:  0.18 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
:: Processing package changes...
reinstalling mondoo...
:: Running post-transaction hooks...
(1/2) Reloading system manager configuration...
  Skipped: Current root is not booted.
(2/2) Arming ConditionNeedsUpdate...
[benr@c8e07418b50e ~]$ mondoo version
! can't find any paths for providers, none are configured home-path=/home/benr/.config/mondoo/providers system-path=/opt/mondoo/providers
cnspec 10.1.2 (cbc7e91, 2024-01-30T22:27:15Z)
```

File: packages/archlinux/mondoo/PKGBUILD (hunk 1 above)
Comment (jaym) [reply packages/archlinux/mondoo/PKGBUILD:13]: did you forget to push? It still fails because of it

**Pull request merged**